apps/admin/src/services/notice/api.ts

```diff
@@ -16,3 +21,8 @@ export const postNotice = async ({ title, content }: PostNoticeReq) => {
     const { data } = await maru.post('/notice', { title, content }, authorization());
     return data;
 };
+
+export const putNotice = async (id: number, { title, content }: PutNoticeReq) => {
+    const { data } = await maru.put(`/notice/${id}`, { title, content }, authorization());
+    return { data };
+};
```

Comment:
```suggestion
export const putEditNotice = async (id: number, { title, content }: PutNoticeReq) => {
    const { data } = await maru.put(`/notice/${id}`, { title, content }, authorization());
    return { data };
};
```

putNotice이라고 했을때 어떠한 동작을 하는건지 알기 어렵습니다 ㅠ edit을 붙혀서 더 명확한 네이밍을 만들면 좋을것같아요!